### PR TITLE
Add more wallets & orders.

### PR DESCRIPTION
**Pull request by InTheta**
You need to update coinbase pro websocket client to get the user data.
I have yet to port my code to your library, so I am using 2 different coinbase libraries right now.

This pull request is mainly for you to cherry pick.

I have ported this from my working library, but have yet to test it.

I will upload bitstamp library asap.

### CODE REVIEW

File: src/Crypto.Websocket.Extensions.Core/Wallets/CryptoWallets.cs [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        
+        public CryptoWalletCollectionReadonly GetWallets(string currency)
+        {
+            var wallets = _currencyToWallet
+                .Where(x => x.Value.Currency==currency)
+                .ToDictionary(x => x.Key, y => y.Value);
+            return new CryptoWalletCollectionReadonly(wallets);
+        }
+        
+        public CryptoWalletCollectionReadonly GetWallets(string currency, string exchangeName)
```
Comment (Marfusios) [inline src/Crypto.Websocket.Extensions.Core/Wallets/CryptoWallets.cs]: `CryptoWallets` is already bound to one specific exchange, what is the purpose of the `exchangeName` parameter?

File: src/Crypto.Websocket.Extensions/Crypto.Websocket.Extensions.csproj [hunk 2]
```diff
@@ -33,6 +33,15 @@
...
   <ItemGroup>
+    <ProjectReference Include="..\..\..\..\Crypto.Arbitrage\bitstamp-client-websocket\src\Bitstamp.Client.Websocket\Bitstamp.Client.Websocket.csproj" />
+    <ProjectReference Include="..\..\..\..\Crypto.Arbitrage\coinbasePro-client-websocket\src\CoinbasePro.Client.Websocket\CoinbasePro.Client.Websocket.csproj" />
+    <ProjectReference Include="..\..\..\..\Crypto.Arbitrage\crypto-exchange-extensions\src\Crypto.Exchange.Extensions\Crypto.Exchange.Extensions.csproj" />
     <ProjectReference Include="..\Crypto.Websocket.Extensions.Core\Crypto.Websocket.Extensions.Core.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="Crypto.Websocket.Extensions.Core, Version=2.0.0.0, Culture=neutral, PublicKeyToken=null">
+      <HintPath>..\..\..\..\Crypto.Arbitrage\crypto-websocket-extensions\src\Crypto.Websocket.Extensions.Core\bin\Debug\netstandard2.0\Crypto.Websocket.Extensions.Core.dll</HintPath>
```
Comment (Marfusios) [inline src/Crypto.Websocket.Extensions/Crypto.Websocket.Extensions.csproj]: Could you remove these hints to the path on the local disk?

File: src/Crypto.Websocket.Extensions.Core/Orders/CryptoOrders.cs [hunk 3]
```diff
@@ -10,25 +10,26 @@
 using Crypto.Websocket.Extensions.Core.Utils;
 using Crypto.Websocket.Extensions.Core.Validations;
 
+
 namespace Crypto.Websocket.Extensions.Core.Orders
 {
     /// <summary>
-    /// Orders manager
+    ///     Orders manager
```
Comment (Marfusios) [inline src/Crypto.Websocket.Extensions.Core/Orders/CryptoOrders.cs]: Please avoid changing the existing code as much as possible (also below with order switching).

File: src/Crypto.Websocket.Extensions.Core/Orders/CryptoOrders.cs [hunk 4]
```diff
@@ -164,17 +170,13 @@ public CryptoOrder FindOrder(string id)
...
 
-        /// <summary>
-        /// Find active order by provided client id
-        /// </summary>
-        public CryptoOrder FindActiveOrderByClientId(string clientId)
+        CryptoOrder ICryptoOrders.FindActiveOrderByClientId(string clientId)
         {
-            var item = GetActiveOrders().FirstOrDefault(x => x.Value.ClientId == clientId);
-            return item.Value;
+            throw new NotImplementedException();
```
Comment (Marfusios) [inline src/Crypto.Websocket.Extensions.Core/Orders/CryptoOrders.cs:174]: ?

File: src/Crypto.Websocket.Extensions.Core/Orders/CryptoOrders.cs [hunk 5]
```diff
@@ -239,32 +248,28 @@ private void Subscribe()
...
         {
-            if (orders == null) 
+            if (orders == null)
                 return;
 
-            foreach (var order in orders)
-            {
-                OnOrderUpdated(order);
-            }
+            foreach (var order in orders) OnOrderUpdated(order);
```
Comment (Marfusios) [inline src/Crypto.Websocket.Extensions.Core/Orders/CryptoOrders.cs:251]: Please, don't leave curly brackets or at least put it on the new line

**Review by Marfusios — CHANGES_REQUESTED** [review]

**Comment by InTheta** [comment]
Apologies it was late.
I will should be able to finish and test this weekend.

**Comment by Marfusios** [comment]
@ColossusFX ping when you will be ready

**Comment by InTheta** [comment]
I have added authed rest requests in coinbase lib now.

Just need to finish testing / converting over the weekend.

**Pull request closed without merging**